### DISCUSSION TITLE
Save Issue

### DISCUSSION THREAD

**Discussion by GreLik84** [jgraph/drawio-desktop#1421] [Q&A]
Hi Guys, I just tried draw.io and it was a pleasant experience until I saved the file locally and returned the next day. When I found that the previous saved file was blank, the new file was a folder and .html file. Only the .html file contains data, but I cant edit anything and nothing on the page works I just see my work. when I try to open recent nothing is in there. please help

**Comment by davidjgraph** [comment]
What desktop version did you use? Can you zip and attach what you saved?
